src/components/Screen.scss

```diff
@@ -1,97 +1,134 @@
 @import 'variables.scss';
 
-.bubble-container {
+.screen-container {
     display: grid;
     grid-template-columns: 20vw 20vw 20vw 20vw 20vw;
     grid-template-rows: 20vh 20vh 20vh 20vh 20vh;
     gap: 0px 0px;
     grid-template-areas:
-      "logo yellow . . ."
-      ". violet . . ."
-      ". . photo . ."
-      ". . . green ."
-      ". pink . . .";
-    max-height: 100vh;
+      "personality-logo soft-skills . personality ."
+      ". hard-skills candidate-name . values"
+      ". . candidate-photo . ."
+      "state-of-mind . . ambition ."
+      ". axis-of-progression . . strength";
+    .bubble-text, .bubble-text-flat {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        text-align: center;
+        height: 9vw;
+        width: 9vw;
+        border-radius: 50%;
+        font-family: roboto;
+        font-weight: 700;
+        font-size: 1.5vw;
+        margin: 1.5vw;
+    }
+    .personality-logo {
+        grid-area: personality-logo;
+        height: 5vw;
+        margin: 2vw;
+    }
+    .candidate-name {
+        grid-area: candidate-name;
+        background-color: $violet-color;
+        color: whitesmoke;
+        position: relative;
+        top: 10vh;
```

Comment:
10vh is big for a relative position: isn't it a layout problem? to be solved with flex/grid?